filament/backend/src/opengl/GLUtils.h

```diff
@@ -452,6 +452,34 @@ constexpr /* inline */ GLenum getInternalFormat(backend::TextureFormat format) n
             return 0;
 #endif
 
+#if defined(GL_EXT_texture_compression_rgtc)
```

Comment:
Updated. I didn't add a flag for WebGL, because unlike S3TC, there doesn't seem to be RGTC and BPTC extensions with `WEBGL_` prefix (they use the `EXT_` prefix instead, just like OpenGL and OpenGLES).

Changed files:
 - OpenGLContext.h/.cpp
 - OpenGLDriver.cpp
 - DriverEnums.h